### ISSUE TITLE
gr.Interface.load does not work with text-to-speech models

### ISSUE THREAD

**Issue by NimaBoscarino** [gradio-app/gradio#1670]
### Describe the bug

Creating a Gradio app with `gr.Interface.load` using a text-to-speech model from Hugging Face will (it seems) download the model and initialize the interface, but attempting to run an inference will throw an error. The error is identical for all the models I tried, which are the first ones among [the top downloaded](https://huggingface.co/models?pipeline_tag=text-to-speech&sort=downloads). I have not yet tried running the models on their own, or with Gradio _without_ relying the `.load()` method. Other model types don't seem to be affected (I tried out sentence-similarity, image-classification, and some other ones).

I reproduced the bug on both Colab and on my local machine, and it looks like [it had worked in the past.](https://youtu.be/k8sHYMeDitQ?t=2982)

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

https://colab.research.google.com/drive/1WPpgeGoItS_CQ7S0L0anJR31LTxqNXCR?usp=sharing

### Logs

```shell
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/gradio/routes.py", line 256, in run_predict
    fn_index, raw_input, username, session_state
  File "/usr/local/lib/python3.7/dist-packages/gradio/blocks.py", line 546, in process_api
    predictions, duration = await self.call_function(fn_index, processed_input)
  File "/usr/local/lib/python3.7/dist-packages/gradio/blocks.py", line 462, in call_function
    block_fn.fn, *processed_input, limiter=self.limiter
  File "/usr/local/lib/python3.7/dist-packages/anyio/to_thread.py", line 32, in run_sync
    func, *args, cancellable=cancellable, limiter=limiter
  File "/usr/local/lib/python3.7/dist-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.7/dist-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.7/dist-packages/gradio/interface.py", line 515, in <lambda>
    if len(self.output_components) == 1
  File "/usr/local/lib/python3.7/dist-packages/gradio/interface.py", line 731, in run_prediction
    pred
  File "/usr/local/lib/python3.7/dist-packages/gradio/components.py", line 2122, in deserialize
    file = processing_utils.decode_base64_to_file(x["data"])
TypeError: string indices must be integers
```

### System Info

```shell
Gradio 3.0.21, Google Colab
```

### Severity

annoying

**Comment by freddyaboulton** [comment]
Thank you for filing @NimaBoscarino ! This looks like the same issue as #1663 . The fix just got merged in #1667 . I'll see if we can do a release this week to get this out ASAP.

**Comment by NimaBoscarino** [comment]
I need to get better at searching the existing issues!

**Comment by freddyaboulton** [comment]
Not a problem at all! Fix is out in version 3.0.22 btw!
